alicloud/data_source_alicloud_cen_bandwidth_packages.go

```diff
@@ -0,0 +1,318 @@
+package alicloud
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/cbn"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAlicloudCenBandwidthPackages() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAlicloudCensBandwidthPackagesRead,
+
+		Schema: map[string]*schema.Schema{
+			"instance_ids": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				ForceNew: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: validateAllowedStringValue([]string{
+					string(InUse),
+					string(Idle),
```

Comment:
I think status should not be input because of if one `Idle` package is used to attach and then the second `terraform` will do not work.